tests/db_functions/datetime/test_extract_trunc.py

```diff
@@ -370,6 +370,14 @@ def test_extract_iso_year_func_boundaries(self):
             (week_1_day_2014_2015, 2015),
             (week_53_day_2015, 2015),
         ], lambda m: (m.start_datetime, m.extracted))
+        # Lookup by iso_year includes date in 2014, when week is in 2015 according to ISO
+        qs = DTModel.objects.filter(start_datetime__iso_year=2015).annotate(
+            extracted=ExtractIsoYear('start_datetime'),
+        ).order_by('start_datetime')
+        self.assertQuerysetEqual(qs, [
+            (week_1_day_2014_2015, 2015),
+            (week_53_day_2015, 2015),
+        ], lambda m: (m.start_datetime, m.extracted))
```

Comment:
There is no need to use `assertQuerysetEqual`, I would compare list of instances, e.g.:
```python
obj_1_iso_2015 = self.create_model(week_1_day_2014_2015, end_datetime)
obj_2_iso_2015 = self.create_model(week_53_day_2015, end_datetime)
...
self.assertSequenceEqual(qs, [obj_1_iso_2015, obj_2_iso_2015])
```

Reply:
done: ba1ed669cadb8db09b66a2f0f779587d5bcd157f